plugins/source/azure/resources/services/security/regulatory_compliance_standards.go

```diff
@@ -30,9 +32,22 @@ func fetchRegulatoryComplianceStandards(ctx context.Context, meta schema.ClientM
 	for pager.More() {
 		p, err := pager.NextPage(ctx)
 		if err != nil {
-			return err
+			// check if we encountered err we can skip
+			var respErr *azcore.ResponseError
+			if !errors.As(err, &respErr) {
+				return err
+			}
+
+			const noStdPricingBundleErrorCode = `Subscription with no standard pricing bundle`
+			if respErr.ErrorCode != noStdPricingBundleErrorCode {
+				return err
+			}
+
+			cl.Logger().Warn().Err(err).Msg("skip fetch due to err from Azure")
```

Comment:
Will this log message have the context of which table was skipped? If not maybe this suggestion would be good enough
```suggestion
			cl.Logger().Warn().Err(err).Msg("skipping regulatory compliance standards fetch for subscription due to to error from Azure")
```